pkg/yttlibrary/struct.go

```diff
@@ -29,6 +29,25 @@ var (
 
 type structModule struct{}
 
+func (b structModule) Make(_ *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if len(args) > 0 {
+		return nil, fmt.Errorf("struct: unexpected positional arguments; structs are made from keyword arguments only")
```

Comment:
core.ErrWrapper will add struct.make to the beginning of error, so `struct: ` is redundant.

Reply:
also instead of ; lets use (...)

Reply:
> also instead of ; lets use (...)

What's the motivation for `...` ?

Argument for `;` is that it's grammatically correct. Semicolon ties two sentences together into a single notion. Ellipsis indicate that intermediate text has been elided. In conversation, it conveys spoken word, trailing off... Granted colloquially, `...` has become interface compatible with `;` ... AFAIK.

Reply:
what i meant was `unexpected positional arguments (structs are made from keyword arguments only)`

Reply:
Oh, geez. Yeah, I see the established pattern for using the parenthetical. 👍